articles/cost-management-billing/manage/assign-roles-azure-service-principals.md

```diff
@@ -37,7 +39,14 @@ Here's an example of the application registration page.
 
 ### Find your SPN and tenant ID
 
-You also need the object ID of the SPN and the tenant ID of the app. You need this information for permission assignment operations later in this article.
+You also need the object ID of the SPN and the tenant ID of the app. You need this information for permission assignment operations later in this article. All applications that get registered in AAD, in the tenant, two types of objects get created once the app registration is done:
```

Comment:
For line 42 and others, all mentions of ADD should be replaced with Azure Active Directory.